src/woo_publications/metadata/admin.py

```diff
@@ -70,3 +70,38 @@ def show_actions(self, obj: Theme) -> str:
             '<a href="{}">{}</a>',
             actions,
         )
+
+
+@admin.register(Organisation)
+class OrganisationAdmin(AdminAuditLogMixin, admin.ModelAdmin):
+    list_display = (
+        "naam",
+        "identifier",
+        "oorsprong",
+        "show_actions",
+    )
+    readonly_fields = (
+        "uuid",
+        "oorsprong",
+    )
+    search_fields = (
+        "identifier",
+        "naam",
+    )
+    list_filter = ("oorsprong", "is_actief")
+
+    def has_change_permission(self, request, obj=None):
+        if obj and obj.oorsprong != OrganisationOrigins.custom_entry:
+            return False
+        return True
```

Comment:
Is it still possible to update `is_actief` now?

I'd rather implement the `def get_fields(...)` so that we can explicitly control which fields can be edited. Alternatively, you can use `def get_readonly_fields`